ThreeBotPackages/tfgrid_solutions/tfgrid_solutions/chatflows/your_flist.py

```diff
@@ -133,14 +160,28 @@ def container_ip(self):
     def overview(self):
         self.md_show_confirm(self.user_form_data)
 
-    @j.baseclasses.chatflow_step(title="Payment", disable_previous=True)
+    @j.baseclasses.chatflow_step(title="Volume Payment", disable_previous=True)
+    def volume_pay(self):
+        if self.container_volume_attach:
+            self.volume = j.sal.zosv2.volume.create(
+                self.reservation, self.node.node_id, size=10, type=self.user_form_data["Volume Disk type"]
```

Comment:
suddenly you hardcoded size

Reply:
ill update it to the user's input

Reply:
Fix: https://github.com/threefoldtech/jumpscaleX_threebot/pull/746/commits/fe4df0f9af97f4d798e11a3965e87ed4fcb50752#diff-e27990502904cf31b839b628ec93acc6R168